src/Molecules/ContestComponent/ContestComponenyStyled.js

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components';
+import FONTS from '../../Styles/fontsStyledComponents';
+import COLORS from '../../Styles/varsStyledComponents';
+
+const ContestComponentStyled = styled.div`
+  box-sizing: border-box;
```

Comment:
Now we have box-sizing ing global styles, you can remove it :)

Reply:
done in new commit!